packages/fxa-content-server/app/scripts/lib/survey-filter.js

```diff
@@ -402,11 +403,12 @@ export const createSurveyFilter = (
     if (
       !(
         surveyConfig &&
-        surveyConfig.rate &&
+        parseInt(surveyConfig.rate) > -1 &&
```

Comment:
Did you intend to use `parseInt` instead of `withinRate`? Seems like you could reuse it.

Reply:
I did - I wanted to ensure that the property existed, was numeric, and was 0 or more. _But_ I think this check could just be moved into `withinRate`. Good catch.